server.js

```diff
@@ -3,15 +3,15 @@ var app = express();
 
 app.set('port', (process.env.PORT || 5000));
 
-app.use(express.static(__dirname + '/'));
-app.use(express.static(__dirname + '/deps/'));
+app.use(express.static(__dirname + '/public/'));
+app.use(express.static(__dirname + '/public/deps/'));
```

Comment:
You shouldn't need this second express.static as deps is a subfolder of public already. The public folder becomes the root of your website (aka localhost/) and deps will be accessible at localhost/deps with just line 6.